src/App.jsx

```diff
@@ -0,0 +1,14 @@
+import { Provider } from "react-redux";
+import "./App.css";
+import Greeting from "./components/Greeting";
+import store from "../redux/store";
+
+function App() {
+  return (
+    <Provider store={store}>
+      <Greeting />
+    </Provider>
+  );
+}
+
```

Comment:
- [ ] One of the requirements was to setup the `<Greeting />` component as a route in your `<App />` component. Please implement this requirement 👍